people/xuewulihuameng/page.en.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Xuewulihuameng
+info:
+    location: Chongqing, Yubei
+---
+
+Xuewulihuameng is a transfemale who is gentle, lovely, kind-hearted but somewhat solitary.
+
+She was skilled in drawing, and she prepared a painting to her friend as a present. Although it was after her departure when the painting was sent to her friend.
+
+<PhotoScroll photos={[ '${path}/photos/painting.jpg',]} />  
+
+In these two Zhihu question of [How about the experience of meeting Xuewulihuameng offline?](https://www.zhihu.com/question/269853559) and [How to appraise the seeking death behavior of Xuewuloihuameng?](https://www.zhihu.com/question/268738337), many friends of her expressed their rememberance and said that Lihua was quite tender. Even before her departure, she still consoled and encouraged one of her friends.
+
+She always treated people around her tenderly. However, what she'd done is so cruel to herself. In one silent midnight, she left an timing will and departed quietly. She weared a beautiful dress, and there is still a smile on her face when she was found by others.
+
+May she be treated tenderly in another world and be a happy girl.       
```

Comment:
May she be treated gently in another world and be a happy girl.

Reply:
I found a explaination:

Tender is related to affection and feeling
- a mother's tender touch
- the tenderness between the owner and their pet

Gentle is descriptive of temperment, how someone acts, their behaviour
- his was always gentle and never angry
- the doctor was gentle in treating his patients
- the bad news was gently given


Is that correct?

Reply:
Correct. But we commonly use "gentle" instead of tender. Moreover, the adverb form of tender "tenderly" is rarely used.